src/Entity/OrganizationRepository.php

```diff
@@ -7,4 +7,18 @@
 
 class OrganizationRepository extends EntityRepository
 {
+
+    public function reverseFindByOrganizarionID($org)
```

Comment:
You could return either a `LocalBusiness` or a `Store` and use `instanceof` in the client code.